doc/c_sdk_resource_information.md

```diff
@@ -0,0 +1,38 @@
+# C-SDK Resource Information
+
+Detailed here are the captured the C SDK Resource constraints for each protocol.
+
+All test are executed on an ubuntu 18.04.1 image with GCC version 7.3.0 using C-SDK version 1.2.12.
+
+## On Disk Binary Information
+
+This is the size of the binary built with the c sdk for each protocol.  The appication setups the SDK and sends 1 telemetry message.
```

Comment:
> c sdk [](start = 45, length = 6)

This is the IoT Hub client only, no DPS Client. Correct? We should clarify that and add section for DPS (and maybe keep as TBD or WIP till we get the data).

Reply:
Yes, DPS can be added later.

---
In reply to: [246860132](https://github.com/Azure/azure-iot-sdk-c/pull/802#discussion_r246860132) [](ancestors = 246860132)